src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {

```

Comment:
The class `Rectangle` should not have an empty line at the beginning of its definition. Please remove the empty line to adhere to Java class structure conventions.